api/api.yml

```diff
@@ -85,6 +85,110 @@ paths:
       responses:
         '200':
           description: computation stopped
+  /launcher/app-user/applications:
+    get:
+      tags:
+      - launcher
+      summary: Get applications in cockpit
+      responses:
+        '200':
+          description: Applications list
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ComputationApplication'
+    post:
+      tags:
+      - launcher
+      summary: Add application to cockpit
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ApplicationRequest'
+      responses:
+        '201':
+          description: Application added
+  /launcher/app-user/applications/{id}:
+    delete:
+        tags:
+        - launcher
+        summary: Remove application from cockpit
+        operationId: removeApplicationFromCockpit
+        parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          required: true
+        responses:
+          '200':
+            description: Application removed
+  /launcher/app-user/rack:
+    get:
+      tags:
+      - launcher
+      summary: Get Computation Rack
+      operationId: getComputationRack
+      responses:
+        '200':
+          description: Applications list
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ComputationTask'
+    post:
```

Comment:
Powinno zwracać 201, ale i tak pokrywa się z POST /launcher/app-user/computations  .

Reply:
Nie do końca, tutaj dodaję tylko do computation rack, a tam uruchamiamy

Reply:
To jw. 201

Reply:
Racja